README.md

```diff
@@ -158,6 +162,11 @@ Same as `yarn build:webapp-redux` but it moves the files to the root directory.
 Launches the test runner for the Redux web application.<br>
 See the section about [running tests](#running-tests) for more information.
 
+### `yarn release:webapp-redux`
+
+It launches a release for Webapp Redux application.<br>
+If there is any commit since the last release with `feat` then it will do a minor bump, if `fix` then it will do a patch bump and if a `!` is on the commit message, it will a BREAKING CHANGE and will do a major bump.<br>
```

Comment:
It will has a more readable format with a list

```suggestion
- `fix` will do a patch bump
- `feat` will do a minor bump
- `!` will do a major bump.
```